### PR TITLE
Revise welcome page

### PR DESCRIPTION
**Pull request by nol166**
Add Coder specific links, remove non-working links.

### CODE REVIEW

File: packages/web/webpack.config.js [hunk 1]
```diff
@@ -65,6 +65,7 @@ module.exports = merge(
 			"spdlog": path.join(vsFills, "spdlog.ts"),
 			"native-watchdog": path.join(vsFills, "native-watchdog.ts"),
 			"iconv-lite": path.join(vsFills, "iconv-lite.ts"),
+			"welcome": path.join(vsFills, "welcomePage.ts"),
```
Comment (coderasher) [inline packages/web/webpack.config.js]: The alias will need to match the import, so in this case `vs/workbench/parts/welcome/page/electron-browser/vs_code_welcome_page`.

**Comment by kylecarbs** [comment]
We should be careful about this. Wouldn't want to miss an update on their welcome page. Not sure if theres really a way around that.

**Comment by coderasher** [comment]
Ahh, true, maybe we should patch it after all? I suggested this but the updates are definitely a good reason to go with patching.

**Comment by Multishifties** [comment]
Should we consider using the update modal that we currently have on our hosted version and stick to using that for all coder-related stuff?

**Comment by nol166** [comment]
Closing this one. Going to make a patch.

**Pull request closed without merging**